OpenRA.Game/Graphics/SpriteLoader.cs

```diff
@@ -21,6 +22,12 @@ public interface ISpriteLoader
 		bool TryParseSprite(Stream s, out ISpriteFrame[] frames);
 	}
 
+	public interface ISpriteLoader2 : ISpriteLoader
```

Comment:
Actually, i'm not sure if you actually need the `.map` file here.  That looks to be used only for grouping the tiles together into individual buildings, but that is not something that we do in these sprite parsers (we have a different concept and set of code called "sequences" for that).  See my other comments in the pr about removing that stitching code.